rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
Getter and setter should be implemented for both `book` and `person`. Kindly use `attr_accessor` instead of `attr_reader` to achieve the same.